node/get_price/src/index.js

```diff
@@ -0,0 +1,103 @@
+
+const getPrice= async function (req, res) {
+
+    const AlphavantageAPIkey = req.env.ALPHAVANTAGE_API_KEY;
+    const GoldAPIkey = req.env.GOLD_API_KEY; 
+    const CoinAPIkey = req.env.COIN_API_KEY;
+    const { type } = req.payload.toLowerCase();
```

Comment:
To ensure compatiblity with Appwrite Functions, please pass a JSON string in as payload and parse it in the function like:

```suggestion
  let payload = {};
  try {
    payload = JSON.parse(req.payload)
  } catch (e) {
    res.json({
      success: false,
      message: `Error parsing payload: ${e}`
    })
    return;
  }
  const type = (payload['type'] ?? '').toLowerCase();
```